Runtime/Game/Requests/AssetRequest.cs

```diff
@@ -2,6 +2,7 @@
 using System.Collections.Generic;
 using LootLocker.Requests;
 using System.Linq;
+using UnityEngine.UI;
```

Comment:
This using statement is not necessary I believe?

Reply:
Correct, fixed